finance/proposal-calls/cycle3/bounties.md

```diff
@@ -0,0 +1,53 @@
+- https://github.com/astropy/astropy/issues/4121 - 1600 $ + 400 $ for review
+- https://github.com/astropy/astropy/issues/4610 - 1600 $ + 400 $ for review
+- https://github.com/astropy/astropy/issues/1803 - 3000 $ + 750 $ for review
+- https://github.com/astropy/astropy/issues/1783 - 1600 $  + 400 $ for review
+- https://github.com/astropy/astropy/issues/3214 - 300 $ + 75 $ for review
+- https://github.com/astropy/astropy/issues/3081 - 1600 $ + 400 $ for review
+- https://github.com/astropy/astropy/issues/6848 - 1600 $ + 400 $ for review
+- https://github.com/astropy/astropy/issues/6618 - 1600 $ + 400 $ for review
+- https://github.com/astropy/astropy/issues/6617 - 600 $ + 150 $ for review
+- https://github.com/astropy/astropy/issues/6583 - 3000 $ + 750 $ for review
+- https://github.com/astropy/astropy/issues/6549 - 1600 $ + 400 $ for review
+- https://github.com/astropy/astropy/issues/6548 - 1600 $ + 400 $ for review
+
+(please suggest further issues for this list during the discussion phase)
+
+Bounties should be set high-enough to attract developers to complex, long-standing issues and give a reasonable rate of return. However, it is impossible to know in advance exactly how much work a specific PR will bring, and the point of this funding request is to try out bounties on a number of issues and see if our (i.e. the people contributing to the list above) intuition works.
+
+Bounty hunters are encouraged, but not required, to post a comment at an issue when they start working on it. Whoever authors the PR that closes the issue can submit an invoice to NumFOCUS to claim the bounty; bounties can be split if requested by the author of the PR.
+We anticipate that only a limited number of people will choose to work on this program and that the workload for NumFOCUS and the finance committee to approve and handle the payments is acceptable. However, the total amount of money in this program for this year is small enough, that it can be done even if every bounty goes a new and entirely different person, who has never received funding from NumFOCUS before.
+
+Bounty hunters assume the risk that somebody else might have a PR to close their issue first or that an issue might turn out to be more complicated that expected. The bounty sums are set when this proposal is accepted and will not be adjusted.
+
+We do not want this proposal to increase the workload for volunteer reviewers, so whoever leaves the most substantial review comments, can claim 25% of the bounty sum. We anticipate that it will tyically be clear and uncontroversial who has the right to claim the "review bounty" and that some fraction of reviewers might not claim the reward anyway. However, the point of this program is to test such assumptions in real live. For this year, and this year only, all disputes about who can claim the bounty will be decided the the group of people listed as organizers of this program. 
+
+### Approximate Budget
+
+10000 $ to 20000 $
```

Comment:
Below 10000 $ it makes little sense to set up this program. It needs to be promoted (inside and outside of astronomy) and we want to evaluate feedback to see if this bounty program should be continued next year so we need enough funding for multiple issues.